<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Курсовая</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxSize">
      <property name="prefix">
       <string>Size: </string>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="codeEdit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="inputMask">
         <string>00.00.00</string>
        </property>
        <property name="text">
         <string>09.03.04</string>
        </property>
        <property name="placeholderText">
         <string>Код направления</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="disciplineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Наименование дисциплины</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="fioLineEdit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>ФИО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dateLine">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="inputMask">
         <string>00.00.0000</string>
        </property>
        <property name="text">
         <string>..</string>
        </property>
        <property name="placeholderText">
         <string>Дата</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSetSize">
      <property name="text">
       <string>Set size</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonInsert">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReadFile">
        <property name="text">
         <string>Read file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonWriteFile">
        <property name="text">
         <string>Write file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
